deploy.sh

```diff
@@ -0,0 +1,3 @@
+oc apply -f tasks/sanity-inspect-image.yaml
```

Comment:
I'm guessing this script is meant for local testing, so it can be omitted from the PR.

Reply:
Yes! Thanks. Sorry, let me remove it.